Formula/libdatrie.rb

```diff
@@ -0,0 +1,36 @@
+class Libdatrie < Formula
+  desc "Double-Array Trie Library"
+  homepage "https://linux.thai.net/~thep/datrie/datrie.html"
+  url "https://github.com/tlwg/libdatrie/releases/download/v0.2.13/libdatrie-0.2.13.tar.xz"
+  sha256 "12231bb2be2581a7f0fb9904092d24b0ed2a271a16835071ed97bed65267f4be"
+  license "LGPL-2.1-or-later"
+
+  def install
+    ENV.deparallelize
+
+    args = std_configure_args
+    args.delete("--disable-debug")
+
+    system "./configure", *args, "--disable-silent-rules"
```

Comment:
```suggestion
    system "./configure", *std_configure_args.delete("--disable-debug"), "--disable-silent-rules"
```
Could you add a comment on why we're deleting `--disable-debug`?

Reply:
It's unsupported by the bundled `./configure`. I'll add a note 👍

Reply:
46acaeb3202